wiki/People/Featured_Artists/en.md

```diff
@@ -9,13 +9,13 @@ tags:
 
 The website's [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) showcases each involved artist. Each artist's page houses general information about their music and a list of their tracks licensed for use in osu! accompanied by pre-timed `.osz` files for beatmapping.
 
-## Selection
+The [osu! team](/wiki/People/osu!_team) sometimes collaborates with artists to create new tracks for use in osu!, called [osu! originals](/wiki/osu!_originals). These are labelled as such in the Featured Artist catalogue.
 
-Recent additions to the Featured Artist library are selected indirectly by the osu! community. If beatmappers create and rank maps for an artist, the [osu! team](/wiki/People/osu!_team) may reach out to the artist to discuss the Featured Artist program. However, some artists are occasionally invited to become Featured Artists based on other factors, such as their expected appeal to osu!'s beatmapping audience.
+## Selection
 
-Most songs in the Featured Artist program are non-exclusively licensed for osu!. In rare cases an artist will create tracks exclusively for osu!, which are labelled with *osu! exclusive* on their Featured Artist listing.
+Additions to the Featured Artist library are selected indirectly by the osu! community. If beatmappers create and rank maps for an artist, the osu! team may reach out to the artist to discuss the Featured Artist program. However, some artists are occasionally invited to become Featured Artists based on other factors, such as their expected appeal to osu!'s beatmapping audience.
```

Comment:
"beatmappers" is rather uncommon as opposed to just "mappers", better rephrase that sentence

Reply:
?

```suggestion
Additions to the Featured Artist library are selected indirectly by the osu! community. If mappers create and rank maps for an artist, the osu! team may reach out to the artist to discuss the Featured Artist program. However, some artists are occasionally invited to become Featured Artists based on other factors, such as their expected appeal to osu!'s mapping audience.
```

Reply:
there isn't a single repeated word in that phrase.. ? <.<